lib/src/page_list_viewport_gestures.dart

```diff
@@ -415,7 +543,7 @@ class DeprecatedPanAndScaleVelocityTracker {
       }
     }
 
-    if (_previousGesturePointerCount! > 1) {
+    if (_previousPointerCount > 1) {
```

Comment:
I can't find where this property is declared, but is there a reason that the name was changed?